src/elements/ChrPlasmaLens.H

```diff
@@ -265,7 +318,9 @@ namespace impactx::elements
         // see: compute_constants() to refresh
         amrex::ParticleReal m_slice_ds;  //! m_ds / nslice();
         amrex::ParticleReal m_beta;
+        amrex::ParticleReal m_gamma;
```

Comment:
Should these be T_Real as well? I'm not really sure about the difference vs amrex::ParticleReal.

This method gets called pre-tracking, and the variables get used in the tracking calculations.